api/dp-openapi.yaml

```diff
@@ -1020,6 +1189,69 @@ components:
       items:
         type: string
 
+    proof_page_text:
+      type: object
+      properties:
+        text:
+          description: the page text from current round.
+          type: string
+
+    proof_page_data:
```

Comment:
Consider calling this `proof_page` to mirror `project_page` -- `data` doesn't really add anything here.

Reply:
done this